css/style.css

```
@@ -16,4 +15,98 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav {
+  padding: 20px 40px 3px 60px;
+  background-color: rgb(247, 244, 241);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+.nav_link > img {
+  width: 50px;
+  height: 40px;
+}
+
+.nav_link ul,
+li {
+  display: flex;
+  font-size: 0.8rem;
+  padding-left: 15px;
+}
```

```suggestion
.nav_link li {
  display: inline;
  font-size: 0.8rem;
  padding: 1rem;
}
```
Here, it looks like we're making the header links next to each other horizontally. Instead, we can make them `inline` boxes as we just want them next to each other